Added link to app on GH pages (#39)
NLESC-JCER/run-cpp-on-web: run-your-c++-code-on-the-web/README.md

> _The resulting page if everything works._
> 
> If you can not or do not want to host the app yourself then visit a version hosted on [GitHub pages](https://nlesc-jcer.github.io/run-cpp-on-web/run-your-c++-code-on-the-web/index.html).

Reviewer: ```suggestion
_The resulting page if everything works. See the live version of the app on [GitHub pages](https://nlesc-jcer.github.io/run-cpp-on-web/run-your-c++-code-on-the-web/index.html)._
```

Author: Nice suggestion, making it part of caption for #34 .1
